crates/router/src/core/customers.rs

```diff
@@ -761,11 +753,7 @@ trait CustomerDeleteBridge {
     ) -> errors::CustomerResponse<customers::CustomerDeleteResponse>;
 }
 
-#[cfg(all(
-    any(feature = "v1", feature = "v2"),
-    not(feature = "customer_v2"),
-    not(feature = "payment_methods_v2")
-))]
+#[cfg(feature = "v1")]
```

Comment:
Same here?

Or are we planning to remove `customer_v2` feature as well in the same PR?